backend/src/main/kotlin/hu/bme/sch/cmsch/component/debt/ProductEntity.kt

```diff
@@ -86,8 +86,10 @@ data class ProductEntity(
     @ColumnDefault("'payments'")
     @Column(nullable = false, length = 255)
     @field:JsonView(value = [ Edit::class, Preview::class, FullDetails::class ])
-    @property:GenerateInput(order = 8, label = "Material Ikon",
-        note = "Innen kell kimásolni a nevét az ikonnak: https://fonts.google.com/icons")
+    @property:GenerateInput(order = 8, label = "React Icons - Fontawesome 5 ikon név",
+        note = "Innen kell kimásolni a nevét az ikonnak:\n" +
+                "https://react-icons.github.io/react-icons/icons/fa\n" +
+                "Ajánlott ikonok: FaHamburger, FaHotdog, FaTshirt, FaPizzaSlice, FaCocktail, FaMoneyBill, FaUtensils, FaUtensilSpoon, FaVial, FaBox, FaBeer, FaHandMiddleFinger, FaShapes, FaHeart, FaFrog")
     @property:GenerateOverview(visible = false)
     @property:ImportFormat(ignore = false, columnId = 6)
     var materialIcon: String = "payments"
```

Comment:
Is this field used in the frontend?
It seems to be used by the admin panel, like this: 
```html
<td><span class="material-icons" th:text="${item.materialIcon}">fastfood</span></td>
```

Reply:
This one is from an addon, that we possible never use again. So I think we can merge this PR